pkg/admissioncontroller/webhook/auth/shoot/shoot_suite_test.go

```diff
@@ -0,0 +1,17 @@
+// SPDX-FileCopyrightText: SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package shoot_test
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+)
+
+func TestShootAuthorizer(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "AdmissionController Webhook Auth ShootAuthorizer Suite")
```

Comment:
Usually, the suite name reflects the package/directory hierarchy. Here `shoot` is replaced with `ShootAuthorizer`. Is that intentional. Could you please share your reasoning?